setup.py

```diff
@@ -64,7 +64,7 @@
     'Flask-Assets>=0.11,<0.12',  # 0.12 will install Flask command 'assets'
     'Flask-Collect==1.2.2',
     'node-semver>=0.1.1',
-    'webassets>=0.11.1'
+    'webassets>=0.11.1,<0.12'
```

Comment:
Can you add a comma at the end of list. Thanks!

--
I know it was previously missing.